### ISSUE TITLE
Integrate into Angular 17 project without standalone components

### ISSUE THREAD

**Issue by micobarac** [valor-software/ng2-charts#1870]
How to integrate the newest version into Angular 17 project without standalone components, using modules? 

`Cannot declare 'BaseChartDirective' in an NgModule as it's not a part of the current compilation.`

`'imports' is only valid on a component that is standalone.`

**Comment by andregreeff** [comment]
hey @micobarac, I actually ran into the same situation just this morning..

I feel like the docs for this project assume the reader is more familiar with Angular itself. basically, if you can't mark the component that renders your chart as "standalone" (see Ng docs for more), then you still need to ensure that you import the required bits yourself.

tl;dr: where the `README` covers the [manual installation steps](https://github.com/valor-software/ng2-charts/blob/master/README.md#manual-install-through-package-managers), in place of step 3 just add `BaseChartDirective` to the `imports` array of your component's parent module.

hope that helps a bit?
